get-started/windows/first-steps-nuget.md

```diff
@@ -0,0 +1,87 @@
+
+## Step 2: Download Telerik UI for .NET MAUI
+
+Telerik UI for .NET MAUI enables you to download the suite either from the Telerik UI for .NET MAUI product page or through your Telerik account. For the purposes of this tutorial, let's download the batch from your Telerik account:
+
+1. Log into your [Telerik Account](https://www.telerik.com/account/).
+
+1. Click the __Downloads__ tab.
+
+  ![Telerik UI for .NET MAUI Download tab in your account](../../images/download-tab.png)
+
+1. Search for MAUI and select the __Telerik UI for .NET MAUI__ product title.
+
+  ![Telerik UI for .NET MAUI Search field in your account](../../images/search-for-maui.png)
+
+1. On the next page, download the `.msi` and `.pkg` automatic installation files, and the Telerik .NET MAUI NuGet Package.
+
+  ![Telerik UI for .NET MAUI available product files in your account](../../images/product-files.png)
+
+## Step 3: Add the Telerik NuGet Package Source to Visual Studio
+
+Now, let's add the Telerik UI for .NET MAUI package through the Telerik NuGet feed. To use the available packages, you need to have an active Telerik account and to authenticate.
+
+1. In Visual Studio, select **Tools** > **NuGet Package Manager** > **Package Manager Settings**.
+
+  ![Telerik NuGet Package Manager context menu with the Package Manager Settings option](../../installation/images/nuget-vs-pm-settings.png)
+
+1. Select **Package Sources**.
+
+  ![Package Sources dialog with the Available package sources field](../../installation/images/nuget-vs-add-source.png)
+
+1. In the **Source** field, add the Telerik server by filling in its [`https://nuget.telerik.com/v3/index.json`](https://nuget.telerik.com/v3/index.json) URL. Click **Update**.
+
+  Note that the [`https://nuget.telerik.com/v3/index.json`](https://nuget.telerik.com/v3/index.json) server will be deprecated and you are recommended to switch to the v3 API, which is faster, more lightweight, and also reduces the number of requests from NuGet clients.
+
+  ![Package Sources field with the checked Telerik NuGet option](../../installation/images/nuget-vs-telerik-server.png)
+
+The Telerik server is now ready to use. You can go to your solution and open the **Solution Package Manager**.
+
+## Step 4: Install the Telerik .NET MAUI NuGet Package
+
+Now, you need to add the Telerik package to the .NET MAUI solution project that you created:
+
+1. Select the Telerik NuGet server as a package source and enter your credentials when prompted.
+1. Search for the `Telerik.UI.for.Maui` package and select it.
+1. Choose the projects which require the package.
+1. Select the desired version and click **Install**.
+
+![Manage Packages for Solutions dialog with the search field and the Telerik.UI.for.MAUI package](../../installation/images/maui-nuget.png)
```

Comment:
This image is taken with a dark theme while the previous images are taken with a light theme. It looks somewhat inconsistent at first glance.